specification/trace/semantic_conventions/http.md

```diff
@@ -153,6 +153,16 @@ This span type represents an outbound HTTP request. There are two ways this can
 
 For an HTTP client span, `SpanKind` MUST be `Client`.
 
+> **Note**
+> It is nice for HTTP client spans to represent the complete lifetime of the HTTP request.
+> An HTTP request ends once the response body has been fully read or the response closes with an error.
+> However, if there is any possibility for application code to not fully read the HTTP response
+> (and for the HTTP client library to then have to clean up the HTTP response asynchronously),
+> the HTTP client span SHOULD be ended earlier (e.g. after the HTTP response headers are read, or fail to be read).
```

Comment:
I want to call out one thing for all the reviewers - I guess as of today we've never put any normative requirements (e.g. SHOULD/MUST) in notes for the API/SDK specs, is my understanding correct? What do we think about having normative requirements in notes?

Reply:
ya, I agree this is odd, I've removed the note and moved it to a separate section